examples/LightStepTestUI/Pods/lightstep/LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright LightStep (http://lightstep.com/)
```

Comment:
I'm truly not sure whether the example's Pod deps should be checked in or not... checking them in makes the example-builds easier (which is why I've done it), but also creates duplication.